Cargo.toml

```diff
@@ -10,7 +10,14 @@ keywords = ["mdns", "discovery", "service-discovery", "zeroconf", "dns-sd"]
 categories = ["network-programming"]
 description = "mDNS Service Discovery library with no async runtime dependency"
 
+[features]
+async = ["flume/async"]
+default = ["async"]
+
 [dependencies]
-crossbeam-channel = "0.5.1" # channel between threads
 log = "0.4.14"              # logging
 nix = "0.23.0"              # socket APIs
+
+[dependencies.flume]
```

Comment:
Just to have a consistent style, could you please define this in the `[dependencies]` section above?